hexa/user_management/models.py

```diff
@@ -53,6 +53,9 @@ def create_superuser(self, email=None, password=None, **extra_fields):
             raise ValueError(_("Superuser must have is_superuser=True."))
         return self.create_user(email, password, **extra_fields)
 
+    def get_by_natural_key(self, email: str):
```

Comment:
Are you sure this override is necessary?

The code from the BaseUserManager in Django is

```python
def get_by_natural_key(self, username):
    return self.get(**{self.model.USERNAME_FIELD: username})
```

It should work for us now?

Reply:
Oh. Yes, you're right. I've missed this one